src/NativeModules/Kit/KitModule.cs

```diff
@@ -39,20 +39,30 @@ public class KitModule : NativeModule {
             Instance = this;
 
             KitManager = new KitManager();
-            KitManager.Load();
 
-            Logger.LogInfo($"Loaded {KitManager.Count} kits");
+            Level.onPostLevelLoaded += Instance.onPostLevelLoaded;
+            if (Level.isLoaded) {
+                onPostLevelLoaded(420);
```

Comment:
What is this for?

Reply:
If uEssentials has been reloaded it will call onPostLevelLoaded (since the level is already loaded the event will not be raised) and load the kitmanager again. In the case that the server is starting for the first time, this will cause kits to load after workshop content is loaded.

Reply:
I see, thanks for the clarification!

One more thing, what does that number (420) means?

Reply:
No problem! 420 is just a placeholder since the onPostLevelLoaded delegate requires an integer parameter. We don't actually use the parameter so it makes no difference what it is.

Reply:
Ok, thanks.